components/kyma-environment-broker/internal/process/provisioning/event_hub_step.go

```diff
@@ -68,6 +68,11 @@ func (p *ProvisionAzureEventHubStep) Run(operation internal.ProvisioningOperatio
 	}
 	log.Infof("HAP lookup for credentials to provision cluster for global account ID %s on Hyperscaler %s", pp.ErsContext.GlobalAccountID, hypType)
 
+	// validate length of name parameter
+	if err = validateLengthOfName(pp.Parameters.Name); err != nil {
```

Comment:
Can't we move the validation to the synchronous http request instead of having it in the asynchronous step? That way, the user would get direct feedback.

Reply:
+1 to @nachtmaar , can we propose this in broker schema?

Reply:
Making this part of the synchronous step sounds good. How about adding this part of already existing validations [here](https://github.com/kyma-incubator/compass/blob/master/components/kyma-environment-broker/internal/broker/instance_create.go#L116)?

Reply:
I think, making this validation as a part of the struct will make things simple and concise.
```Name                        string   `json:"name" validate:"required,min=6"```

Reply:
I'd say, we'd also need to set the constraint on the graphQl layer over here: https://github.com/kyma-incubator/compass/blob/master/components/kyma-environment-broker/internal/broker/plans.go#L155

Reply:
> Making this part of the synchronous step sounds good. How about adding this in the part of already existing validations here? 
will this not throw runtime panic in case of unmarshalling? @sayanh

Reply:
Hi, I'm ok with adding that to the schema as @anishj0shi proposed 👍

Reply:
Changed in JSON schema.